src/drawer.js

```diff
@@ -423,6 +423,24 @@ $.Drawer.prototype = {
             this.context.globalCompositeOperation = compositeOperation;
         }
         if (bounds) {
+            // Internet Explorer and Microsoft Edge throw IndexSizeError 
+            // when you call context.drawImage with negative x or y 
+            // or width or height greater than the canvas width or height respectively
+            if (bounds.x < 0) {
+                bounds.width += bounds.x;
+                bounds.x = 0;
+            }
+            if (bounds.width > this.sketchCanvas.width) {
```

Comment:
When using rotation, the sketchCanvas can be bigger than the main canvas.
So I think it would be safer to compare to `this.canvas.width`.